src/main/resources/import.sql

```diff
@@ -83,3 +95,9 @@ INSERT INTO health_referral (health_referral_id, hlkid, mbo, therapy_code) VALUE
 INSERT INTO health_referral (health_referral_id, hlkid, mbo, therapy_code) VALUES ('DGC_167838/015H2463F8T600004D7C', 'HR177695', 901234567, '#9M8D2');
 INSERT INTO health_referral (health_referral_id, hlkid, mbo, therapy_code) VALUES ('DGC_167839/015H2463F8T600004D7D', 'HR219873', 876543210, '#6W1E6'); --inactive
 
+
+INSERT INTO patient (id, date_of_birth, address, mbo, oib, phone_number) VALUES (1, '2000-01-01', 'tesna ulica 1', '0123445289', '127856789', '0911223456');
+INSERT INTO patient (id, date_of_birth, address, mbo, oib, phone_number) VALUES (2, '2003-01-01', 'tesna ulica 4', '0123456789', '123436789', '0911223456');
+INSERT INTO patient (id, date_of_birth, address, mbo, oib, phone_number) VALUES (3, '2002-04-01', 'tesna ulica 2', '0123456489', '123696789', '0911223456');
+INSERT INTO patient (id, date_of_birth, address, mbo, oib, phone_number) VALUES (4, '2004-01-01', 'tesna ulica 1', '0123455889', '121256789', '0911223456');
+INSERT INTO patient (id, date_of_birth, address, mbo, oib, phone_number) VALUES (5, '2001-01-05', 'tesna ulica 8', '0123447789', '124556789', '0911223456');
```

Comment:
izmijeniti mbo i oib da budu drugaciji od drugih jer nas kasnije to moze zeznuti zbog pretrazivanja. MBO i OIB moraju biti jedinstveni